controllers/articles.controller.js

```diff
@@ -21,4 +21,12 @@ exports.getArticleById = (req, res, next) => {
         .catch((err) => {
             next(err); 
         });
-};
\ No newline at end of file
+};
+
+exports.getAllArticles = (req, res, next) => {
+    fetchArticles(req.query)
+      .then((articles) => {
+        res.status(200).send({ articles });
+      })
+      .catch(next); 
```

Comment:
Try to be consistent in your approach to catching errors. 

Your previous controller uses 
```  
  .catch((err) => {
            next(err); 
        }); 
```
Either approach is fine (and equally readable in my opinion) but try to stick with a consistent approach